src/containers/Column/Actions/Actions.js

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes } from 'react'
+import { Button, Dropdown, Icon } from 'semantic-ui-react'
+
+class Actions extends React.Component {
+  render () {
+    const trigger = (
+      <Button basic>
+        <Icon name='ellipsis horizontal' />
+      </Button>
+    )
+    return (
+      <Button.Group floated='right'>
+        <Dropdown
+          button
+          floating
+          icon={null}
+          labeled
+          options={this.props.actions}
+          trigger={trigger}
+        />
+      </Button.Group>
+    )
```

Comment:
Does this need to be a dropdown?, I guess it's better so people don't accidentally delete columns, or cards, but it seems to be redundant for one action at the moment.